src/ethlance/server/db.cljs

```diff
@@ -74,7 +74,7 @@
      ;; PK
      [(sql/call :primary-key :user/address)]
      ;; FKs
-     [[(sql/call :foreign-key :user/address) (sql/call :references :User :user/address)]]]
+     [(sql/call :foreign-key :user/address) (sql/call :references :User :user/address) (sql/raw "ON DELETE CASCADE")]]
```

Comment:
adding all this "on delete cascade" looks to me like we are planning to delete records from our db. Do we have this requirements in ethlance? Or what is its purpose?

Reply:
it's for: https://github.com/district0x/ethlance/blob/graphql/resolvers/src/ethlance/server/db.cljs#L470